### PR TITLE
add footer and modify styles

### PR DESCRIPTION
**Pull request by Gblur**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,7 +29,13 @@ <h1 class="header-title">Quiz App</h1>
       </div>
     </section>
   </main>
-  <footer></footer>
+  <footer>
+    <nav class="footer-container">
+      <a href=""><i class="fa-solid fa-bookmark"></i></a>
+      <a href=""><i class="fa-solid fa-bookmark"></i></a>
+      <a href=""><i class="fa-solid fa-bookmark"></i></a>
```
Comment (PReichetanz) [inline index.html:36]: I don't see why you've used three `<a>` tags here: they show the same icon and don't link anywhere. We should avoid empty or unused tags at all costs (this is a question of the agile workflow).

File: styles/index.css [hunk 2]
```diff
@@ -58,4 +58,19 @@ h1 {
   background: #c9c9c9;
 }
 
+footer {
+  position: relative;
+}
+
+.footer-container {
```
Comment (PReichetanz) [inline styles/index.css:65]: The name is misleading since it is the `nav` element. What about the class name `navigation` to be precise?

**Review by PReichetanz — COMMENTED** [review]
Nice work, I found two minor issues to be discussed.

File: index.html (hunk 1 above)
Comment (Gblur) [reply index.html:36]: good point.
just for filling the structure and use it as a placeholder first I thought.

File: styles/index.css (hunk 2 above)
Comment (Gblur) [reply styles/index.css:65]: Indeed

**Pull request merged**